src/objecttypes/api/v2/openapi.yaml

```diff
@@ -111,6 +111,25 @@ paths:
   /objecttypes/{objecttype_uuid}/versions:
     get:
       operationId: objectversion_list
+      description: |-
```

Comment:
I don't think these descriptions were added intentionally, could you either remove them or add a more correct description of `ObjectVersion`?

Reply:
See https://github.com/maykinmedia/objecttypes-api/pull/149/commits/2caaf508793063cd8b7bcbf0c60210342f55a355